doc/classes/String.xml

```diff
@@ -747,7 +747,7 @@
 		<method name="reverse" qualifiers="const">
 			<return type="String" />
 			<description>
-				Returns the copy of this string in reverse order.
+				Returns the copy of this string in reverse order. The implementation is straightforward, that means symbols like emoji and compound letters of some scripts will be reversed by codepoints rather than by sequences of codepoints, and the result is usually undesired in that case.
```

Comment:
```suggestion
				Returns the copy of this string in reverse order. This operation works on unicode codepoints, rather than sequences of codepoints, and may break things like compound letters or emojis.
```
IMO this is simpler to parse? If agreed, this goes for both changes, of course. :)

Reply:
Done